WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -219,7 +220,15 @@ private static void TerminateApplication()
 
 	private static void TaskScheduler_UnobservedTaskException(object? sender, UnobservedTaskExceptionEventArgs e)
 	{
-		Logger.LogDebug(e.Exception);
+		// Until https://github.com/MetacoSA/NBitcoin/pull/1089 is resolved.
```

Comment:
Tested, it works the exact same way but IMO looks hackier.

Reply:
Looks hackier in my eyes because of count check and hardcoded `array[0]`😆
I understand your point though, but `AggregateException.Flatten().InnerException` will give back the original exception that caused the whole thing to roll down here, in this example the `SocketException` of an aborted I/O operation. This is what we are looking for and want to avoid, right?
Or the point is that we know that NBitcoin's exception will be an `AggregateException` with 1 inner exc, which is a `SocketException`, so that is the condition that we are looking for, and let any other through to log because then it's ours?
Im not against using your code in any way, im just trying to understand the whys and whens😸